src/Microsoft.Azure.WebJobs.Logging/WebJobs.Logging.csproj

```diff
@@ -1,150 +1,29 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wondered. I can try to remove it, but thought some of the other projects had dependencies on this.